src/main/java/org/jenkins/tools/test/hook/StructsHook.java

```diff
@@ -1,53 +0,0 @@
-package org.jenkins.tools.test.hook;
-
-import hudson.model.UpdateSite;
-import hudson.util.VersionNumber;
-import java.util.Map;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-import org.jenkins.tools.test.model.PomData;
-
-public class StructsHook extends AbstractMultiParentHook {
```

Comment:
Structs was flattened a while ago, and this hook was only needed to test older versions where it is not flattened. Since I do not think there are any consumers left that would be trying to run PCT on old versions of structs, deleting this hook.

Reply:
need to update [this javadoc comment](https://github.com/jenkinsci/plugin-compat-tester/blob/7e9e0f7cc8e0af07f5bc2d90d584416768b49cbc/src/main/java/org/jenkins/tools/test/hook/AbstractMultiParentHook.java#L146)?

Reply:
Sure, why not